WooCommerce/Classes/POS/Presentation/PointOfSaleDashboardView.swift

```diff
@@ -130,7 +136,7 @@ private extension PointOfSaleDashboardView {
     }
 
     var productListView: some View {
-        ItemListView(viewModel: itemListViewModel)
+        ItemListView(viewModel: self.itemListViewModel, dashboardViewModel: viewModel)
```

Comment:
Although we haven't yet found a good approach for our architecture (https://github.com/woocommerce/woocommerce-ios/issues/13441), maybe we could pass [a binding](https://developer.apple.com/documentation/swiftui/binding#overview) here rather than a whole dashboardViewModel? 

The idea being that `itemListViewModel` could change `isShowing = true` and then `PointOfSaleDashboardView` would react accordingly.